src/functions.php

```diff
@@ -47,9 +47,9 @@ function view(string $path): View
         return new GenericView($path);
     }
 
-    function request(string $uri, array $body = [], array $headers = []): Request
+    function request(string $uri, array $body = [], array $headers = []): PsrRequest
```

Comment:
We don't really need this method anymore since we have proper testers. But we'll need to do some refactorings, so I just made it work with PSR for now.